src/rpc/protocol/jsonRpc.ts

```diff
@@ -18,23 +18,26 @@ export class JSONRPC {
   private transport: LikeSocketServer;
   private jsonRpcApi;
 
-  public constructor(opts, {transport, api}: {transport: LikeSocketServer; api: API}) {
+  public constructor(opts, {transport, api}: {transport: LikeSocketServer; api: IBeaconApi | IValidatorApi}) {
```

Comment:
I think we need to pass array in api props because you want those apis exposed on all passed transports

Reply:
future pr, ended up removing the beacon api entirely